docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -41,18 +41,16 @@ Follow the instructions for [GitHub](../repositories-configure/integrations/gith
 
 ![Enabling your Git provider integration](../repositories-configure/integrations/images/github-integration.png)
 
-## 3. Blocking merging pull requests {: id="blocking-pull-requests"}
+## 3. Blocking merging pull requests (optional) {: id="blocking-pull-requests"}
 
-<!--TODO Add discussion about when it's time for customers to actually block merging pull requests-->
-!!! important
-    At this stage we recommend that you:
+Once you've tested out Codacy for a while and you're happy with the level of feedback provided, you can decide to enforce the quality gates and use Codacy to block merging pull requests on your Git provider. This is the best way to protect your code from unwelcome changes, but will also be an integral part of your development pipeline. It's therefore important that before activating this feature, you:
 
-    -   Validate that Codacy is reporting the intended status on your pull requests
-    -   Double check you repository's [tool and code pattern settings](../repositories-configure/configuring-code-patterns.md) and [quality gate settings](../repositories-configure/adjusting-quality-settings.md)
+-   Validate that Codacy is reporting the intended status on your pull requests
+-   Double check you repository's [tool and code pattern settings](../repositories-configure/configuring-code-patterns.md) and [quality gate settings](../repositories-configure/adjusting-quality-settings.md)
 
-    This is important to eliminate any false positives that could inadvertently block the work of your team.
+This is important to eliminate any false positives that could inadvertently block the work of your team.
```

Comment:
There's some repetition between "it's therefore important" and "this is important". One way to improve this could be to mention the false positives directly on the sentence introducing the list, for example:

> It's therefore important to eliminate any false positives that could inadvertently block the work of your team before activating this feature:

Reply:
Now that I see this version, I even think we could bring back the `important` admonition to give more visibility just to this part:

```
!!! important
    It's important to eliminate any false positives that could inadvertently block the work of your team before activating this feature:
    
    - Validate [...]
    - Double-check [...]
```

What do you think?

Reply:
Good idea. I moved the sentences around and reworded them for flow. Let me know 😄